### PR TITLE
feat(ui): Adds component for displaying version names in releasesV2

### PR DESCRIPTION
**Pull request by matejminar**
This PR adds a new component for displaying parsed release version names. 
It is a part of health/releasesV2 initiative and will be replacing `app/components/version.tsx` component in the future.

This also modifies the tooltip component to accept a new prop called `isHoverable` - if true, a user is able to hover tooltip without it disappearing (nice if you want to be able to copy tooltip contents to clipboard).

### CODE REVIEW

File: docs-ui/components/formatters.stories.js [hunk 1]
```diff
@@ -100,4 +102,28 @@ storiesOf('Utility|Formatters', module)
...
+    withInfo('Formats release version')(() => {
+      const version = text('version', 'foo.bar.Baz@1.0.0+20200101');
+      const orgId = text('orgId', 'sentry');
+      const anchor = boolean('anchor', true);
+      const preserveGlobalSelection = boolean('preserveGlobalSelection', false);
+      const tooltipRawVersion = boolean('tooltipRawVersion', true);
+      const className = text('className', 'asdsad');
+      return (
+        <div>
+          {version} =
```
Comment (priscilawebdev) [inline docs-ui/components/formatters.stories.js]: please, is this right?

File: src/sentry/static/sentry/app/components/versionV2.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  anchor?: boolean;
+  /**
+   * Should link to Release preserve user's global selection values
+   */
+  preserveGlobalSelection?: boolean;
+  tooltipRawVersion?: boolean;
+  className?: string;
+};
+
+export const VersionV2 = ({
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/versionV2.tsx]:
```suggestion
const VersionV2 = ({
```

**Comment by priscilawebdev** [comment]
Good Job! 💪

**Review by priscilawebdev — APPROVED** [review]

File: docs-ui/components/formatters.stories.js (hunk 1 above)
Comment (matejminar) [reply docs-ui/components/formatters.stories.js]: Yes, it is like a before/after thing. Other formatter called count does the same thing.
image

Comment (priscilawebdev) [reply docs-ui/components/formatters.stories.js]: Interesting! Thanks for the clarification ;-)

**Pull request merged**